stock_orderpoint_manual_procurement/models/stock_warehouse_orderpoint.py

```diff
@@ -24,14 +24,6 @@ class StockWarehouseOrderpoint(models.Model):
         compute="_compute_procure_recommended",
     )
 
-    def _prepare_procurement_values(self, product_qty, date=False,
```

Comment:
You don't have to remove this either. Please, readd it

Reply:
@lreficent We have added that method to use of product quantity. Afterwards, we improve it at https://github.com/OCA/stock-logistics-warehouse/commit/bcccda5bd4bc4ee60e0d12b81fcb309a231d3af3

So now no need of these method to get product quantity. Can you guide me to further use of it ?

Reply:
Ok, no related to this fix, I got you now. Thanks!